Dockerfile

```diff
@@ -3,12 +3,7 @@ ARG TARGET=server
 # Can be used in case a proxy is necessary
 ARG GOPROXY
 
-# Build Temporal binaries
-FROM golang:1.16-alpine AS builder
-
-RUN apk add --update --no-cache \
-    make \
-    git
+FROM temporalio/base-builder AS temporal-builder
 
 # Making sure that dependency is not touched
 ENV GOFLAGS="-mod=readonly"
```

Comment:
Can you try to remove this? It shouldn't be needed anymore.

Reply:
done

Reply:
Not done!

Reply:
got reverted at some point, done now.